index.html

```diff
@@ -86,12 +90,14 @@
 						<img src="img/km.jpg" alt="Kheng Meng" />
 						<p>khengmeng</p>
 					</li>
+					<li>
+						<img src="img/chuming.jpg" alt="Kheng Meng" />
+						<p>chuming (guest mc)</p>
```

Comment:
Didn't realise that the content was for the previous Hackware! Will change it ASAP.